lib/bureaucrat/markdown_writer.ex

```diff
@@ -198,7 +198,7 @@ defmodule Bureaucrat.MarkdownWriter do
     end)
   end
 
-  defp get_controller({_, opts}), do: opts[:group_title] || strip_ns(opts[:module])
+  defp get_controller({_, opts}), do: opts[:group_title] || String.replace_suffix(strip_ns(opts[:module]), "Test", "")
```

Comment:
what is "Test" ?

Reply:
This will remove the `Test` suffix from the module name `HelloChannelTest` -> `HelloChannel`.